apps/services/auth/delegation-api/test/access-test-cases.ts

```diff
@@ -179,10 +189,16 @@ export const accessTestCases: Record<string, TestCase> = {
         ],
       },
     ],
-    expected: [],
+    expected: [
+      {
+        name: 'd1',
+        scopes: [{ name: 's1' }],
+      },
+    ],
```

Comment:
Did you mean to change this test case? Isn't it valid as it was, for outgoing?

Reply:
This change is on the `unrelatedCustomDelegations` not `customDelegationScopes`, and as I didn't apply any direction on the `unrelatedCustomDelegations` as it is not related to the user, the domain and scope is returned in the corresponding endpoints.

Reply:
I know, but the purpose of this test case is to make sure that unrelated custom delegations don't affect `direction=outgoing` filtering of the current user.

Reply:
The `unrelatedCustomDelegations` is important test case for the delegations controller but causes trouble in the general `it.each` approach for the domains. So I introduced `expectedDomains` property to solve this test case for the domains.